### PR TITLE
CryptoX 1.1.0 and 1.1.1 releases

### PR DESCRIPTION
**Pull request by Radiokot**
## Purpose

Add release notes for 1.1.0 and 1.1.1

## Changes

- Add release notes for 1.1.0 and 1.1.1

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 1]
```diff
@@ -18,77 +18,106 @@ Wallets
...
+    **1.1.1 - June 11, 2024**
 
-    Version 1.0.0 brings feature parity with |mw-gen2|, fixes multiple bugs and improves the overall user experience.
+    Version 1.1.1 improves WalletConnect pairing stability in situations where the connection is poor
+
+    **1.1.0 - June 06, 2024**
+
+    Version 1.1.0 removes shielding, simplifies restoring a phrase-based wallet and fixes some issues.
+
+    What have been removed:
```
Comment (td202) [inline source/mainnet/net/release-notes/release-notes-lp.rst:29]:
```suggestion
    What has been removed:
```

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 2]
```diff
@@ -18,77 +18,106 @@ Wallets
...
+
+    - Shielding – now it is only possible to unshield your balances, while shielding and sending shielded funds are no longer available
 
     The following features have been added:
 
-    - Setting up and updating validator pool commission rates
+    - Ability to unshield your balances from the "More" screen
+
+    - Support for WalletConnect binary messages signing
```
Comment (td202) [inline source/mainnet/net/release-notes/release-notes-lp.rst:37]:
```suggestion
    - Support for signing WalletConnect binary messages
```

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 3]
```diff
@@ -18,77 +18,106 @@ Wallets
...
+
+    - Ability to paste the phrase from the clipboard when importing a wallet
+
+    Other changes:
+
+    - The "Watch video" link on the welcome screen now opens the account creation tutorial
+
+    - Actualized the Terms and the Privacy Policy
```
Comment (td202) [inline source/mainnet/net/release-notes/release-notes-lp.rst:45]: I'm not sure what this means.

**Review by td202 — APPROVED** [review]

File: source/mainnet/net/release-notes/release-notes-lp.rst (hunk 3 above)
Comment (td202) [reply source/mainnet/net/release-notes/release-notes-lp.rst:45]: I meant, what is meant by "actualized"?

Comment (Radiokot) [reply source/mainnet/net/release-notes/release-notes-lp.rst:45]: Will be changed to "Updated"

Comment (td202) [reply source/mainnet/net/release-notes/release-notes-lp.rst:45]:
```suggestion
    - Updated the Terms and the Privacy Policy
```

**Pull request merged**